File: 240621/남녀노소 구분짓기/sex-and-age.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int s = sc.nextInt();
        int age = sc.nextInt();


        if (s == 0 && age >= 19) {
            System.out.println("MAN");
        } else if (age < 19) {
            System.out.println("BOY");
        }
          if (s == 1 && age >= 19) {
            System.out.println("WOMAN");
        } else if (age < 19) {
            System.out.println("GIRL");
        }
        
        // 여기에 코드를 작성해주세요.
    }
}

File: 250306/남녀노소 구분짓기/sex-and-age.js
const fs = require("fs");

let input = fs.readFileSync(0).toString().trim().split("\n");

let sex = Number(input[0]);
let age = Number(input[1]);


if (age >= 19) {
    if (sex === 0) {
        console.log("MAN");
    } else {
        console.log("WOMAN");
    }

} else {
    if (sex === 0) {
       console.log("BOY");
    } else {
         console.log("GIRL");
    }
}
